react/searchResult.css

```diff
@@ -234,6 +231,10 @@
     grid-template-columns: 1fr;
   }
 
+  .galleryRow {
```

Comment:
Why are you defining `.galleryRow` class again? You could just add this to the one in line 93.

Reply:
Ops, it's inside a media query.